services/nostr-band/nostr-band-followers.service.js

```diff
@@ -0,0 +1,57 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { BaseJsonService } from '../index.js'
+
+const npubSchema = Joi.object({
+  followers_pubkey_count: Joi.number().required(),
+}).required()
+
+const mainSchema = Joi.object({
+  stats: Joi.object().pattern(Joi.string(), npubSchema).required(),
+}).required()
+
+export default class NostrBandFollowers extends BaseJsonService {
+  static category = 'social'
+
+  static route = {
+    base: 'nostr-band/followers',
+    pattern: ':npub',
+  }
+
+  static examples = [
```

Comment:
Can we write this as an `openApi` object. See the docs in https://github.com/badges/shields/blob/master/doc/TUTORIAL.md#44-adding-documentation-to-the-frontend

We do still have some services using the old `examples` format, but there are not many left and we are in the process of converting them all. See https://github.com/badges/shields/issues/9285

New badges should use the `openApi` format.

Reply:
done: https://github.com/badges/shields/pull/9870/commits/16a1b31ae17ff9b019875f1e93abc06a9378af55